trackma/utils.py

```diff
@@ -229,6 +230,16 @@ def load_data(filename):
 
 
 def save_data(data, filename):
+    if '.list' in filename:
+        path = expand_path('~/Desktop/Coding_Projects/new-os-setup/linux-backup/anilist.backup')
```

Comment:
This hard-coded past needs to be replaced with a dynamic one if this is meant to stay. Not really sure what the purpose of this JSON backup is, though. Sounds like a better idea for this would be to move the old file to a backup path first and then start writing the main file.